### PR TITLE
[REVIEW] Fix predicate for testing cudaMallocAsync APIs

### PR DESCRIPTION
**Pull request by shwina**

### CODE REVIEW

File: python/rmm/tests/test_rmm.py [hunk 1]
```diff
@@ -23,6 +23,9 @@
 
 _driver_version = rmm._cuda.gpu.driverGetVersion()
 _runtime_version = rmm._cuda.gpu.runtimeGetVersion()
+_CUDAMALLOC_ASYNC_SUPPORTED = (_driver_version >= 11020) and (
```
Comment (galipremsagar) [inline python/rmm/tests/test_rmm.py:26]: It has to `or` no(instead of `and`)?

Comment (shwina) [reply python/rmm/tests/test_rmm.py:26]: We need both conditions to hold, correct?

Comment (galipremsagar) [reply python/rmm/tests/test_rmm.py:26]: Ah, never mind didn't notice `>=`

**Review by galipremsagar — APPROVED** [review]

File: python/rmm/tests/test_rmm.py (hunk 1 above)
Comment (shwina) [reply python/rmm/tests/test_rmm.py:26]: Note this variable indicates whether `cudaMallocAsync` _is_ supported.

Comment (galipremsagar) [reply python/rmm/tests/test_rmm.py:26]: > Note this variable indicates whether `cudaMallocAsync` _is_ supported.

Yup, missed this.

**Pull request closed without merging**